scripts/coverage-report.sh

```diff
@@ -13,14 +13,14 @@ app_name=talos
 export CARGO_INCREMENTAL="0"
 export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort"
 
-excludes="none"
+excludes="cohort_sdk_js"
```

Comment:
Had to add it to avoid failing build here in Github. Apparently Scylla has similar issue for which there is no solution yet.